extensions/grpc/pom.xml

```diff
@@ -20,9 +20,6 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>hazelcast-jet-grpc</artifactId>
-    <properties>
-        <guava.version>28.1-android</guava.version>
```

Comment:
I'm not sure why it was an android version used here but maybe it does matter?

Reply:
I don't recall any particular reason for this, if the tests in the grpc module pass it should be ok.